### PR TITLE
Makes eyestabbing and neck slicing a little easier to use. Also add's screams to kneecapping.

### PR DESCRIPTION
**Pull request by Helg2**
## About The Pull Request
Neck slicing warning is completely useless and annoying, so i deleted it. Everytime i sliced someone's neck i needed to turn off combat mode just to continue killing them which is annoying as hell.

Eye stabs warnings are too completely useless and annoying. If you're in the fight you definitely don't want to read warnings about them being alien or their eyes being covered.
Also made it's code a little bit prettier?

Made the victim's of kneecapping scream on act. Everything for the sadistic pleasurement!
## Why It's Good For The Game
Easier to fight with those things. More fun.
## Changelog
:cl:
add: the victims of kneecapping will now scream.
qol: you will no longer gain useless message, when you're harming someone in agro-grab in the head with their neck already slicen, instead of continuing killing them.
qol: you will no longer gain irritating messages, when trying to stab someone in the eye, instead of just stabbing them with screwdriver.
/:cl:

### CODE REVIEW

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/datums/elements/eyestab.dm [hunk 1]
```diff
@@ -40,39 +40,33 @@
...
-		to_chat(user, span_warning("You failed to stab [target.p_their()] eyes, you need to remove [target.p_their()] eye protection first!"))
-		return
-
-	if (isalien(target))
-		to_chat(user, span_warning("You cannot locate any eyes on this creature!"))
-		return
-
 	if (isbrain(target))
 		to_chat(user, span_warning("You cannot locate any organic eyes on this brain!"))
 		return
```
[the remark this answers is not in the document]
Comment (Helg2) [reply code/datums/elements/eyestab.dm:61]: I'm not sure if you can attack brains, so i left it here xd. I'll check it in a min.

[the remark this answers is not in the document]
Comment (Helg2) [reply code/datums/elements/eyestab.dm]: Okay, didn't know.

[the remark this answers is not in the document]
Comment (Helg2) [reply code/datums/elements/eyestab.dm]: Thinking about it, does this part has it's use at the moment even?

File: code/datums/elements/kneecapping.dm [hunk 2]
```diff
@@ -64,6 +64,10 @@
 	if(!leg)
 		return
 
+	for(var/datum/wound/blunt/wound in leg.wounds)
+		if(wound.severity == WOUND_SEVERITY_CRITICAL)
+			return
```
[the remark this answers is not in the document]
Comment (Helg2) [reply code/datums/elements/kneecapping.dm:69]: I'll try to add just the screams if they are already broken then.

[the remark this answers is not in the document]
Comment (Helg2) [reply code/datums/elements/kneecapping.dm]: Added

**Comment by Helg2** [comment]
Can someone remove the "Fix" and add the "Feature" then please?

**Review by Jacquerel — CHANGES_REQUESTED** [review]
One nitpick then we're good to go.

**Pull request merged**